test/JDBC/input/BABEL-1465-vu-verify.sql

```diff
@@ -0,0 +1,229 @@
+-- CAST from char to varchar is immutable/deterministic and can be used to generate computed column
+alter table babel_1465_vu_prepare_t1 add b as cast(cast('01-01-2012' as char(10)) as varchar(10)) persisted;
```

Comment:
why these alters in verify?

Reply:
We are trying to verify if these functionalities work and the behaviour is same even after the upgrade, For example volatile functions should not be allowed to be added as computed columns